calicoctl/commands/node/status.go

```diff
@@ -129,7 +136,19 @@ func drawTable(birdOut string) {
 
 		if s != "" {
 			col := []string{}
-			fields := strings.Fields(line)[3:6]
+			f := strings.Fields(line)[3:]
```

Comment:
I think the problem here is more with this line:
```
col = append(col, fields...)
```

Where we're assuming a set number of fields.

We should probably be explicitly adding in the fields we expect:

```
// Add State, Since and Info columns.
col = append(col, fields[0])
col = append(col, fields[2])
col = append(col, fields[3]) 
```